tests/test_ftrace.py

```diff
@@ -385,7 +385,7 @@ def test_parse_tracing_mark_write_events(self):
         # put under trace.cpu_frequency
         self.assertEquals(trace.tracing_mark_write.data_frame.iloc[-1]["string"],
                           "TRACE_MARKER_START")
-        self.assertEquals(len(trace.tracing_mark_write.data_frame), 1)
```

Comment:
No, the code and the comment are already aligned.  The comment says that there should be only one event in `tracing_mark_write` because the second one should be in `cpu_frequency` and this test checks that it is true

Reply:
Ok, I'll add the check on both then, since we also want to make sure that cpu_frequency got its item.